Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -15,6 +15,16 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
 
     public init() {}
 
+    public var flexibleRightSpacing = false
+
+    public init(configuration: AnyObject) throws {
+        flexibleRightSpacing = configuration["flexible_right_spacing"] as? Int == 1
```

Comment:
Uhm interesting. 
`as? Bool == true` works OK now. 

Adding another commit to modify this.